yt_dlp/extractor/nhk.py

```diff
@@ -45,24 +47,32 @@ def _get_api_info(self, refresh=True):
         self.cache.store('nhk', 'api_info', api_info)
         return api_info
 
-    def _extract_formats_and_subtitles(self, vod_id):
+    def _extract_stream_info(self, vod_id):
         for refresh in (False, True):
             api_info = self._get_api_info(refresh)
             if not api_info:
                 continue
 
             api_url = api_info.pop('url')
-            stream_url = traverse_obj(
-                self._download_json(
-                    api_url, vod_id, 'Downloading stream url info', fatal=False, query={
-                        **api_info,
-                        'type': 'json',
-                        'optional_id': vod_id,
-                        'active_flg': 1,
-                    }),
-                ('meta', 0, 'movie_url', ('mb_auto', 'auto_sp', 'auto_pc'), {url_or_none}), get_all=False)
+            meta = traverse_obj(self._download_json(api_url, vod_id, 'Downloading stream url info',
+                                fatal=False, query={
+                                    **api_info,
+                                    'type': 'json',
+                                    'optional_id': vod_id,
+                                    'active_flg': 1, }), ('meta', 0))
+
+            stream_url = traverse_obj(meta, ('movie_url', ('mb_auto', 'auto_sp', 'auto_pc'),
+                                      {url_or_none}), get_all=False)
```

Comment:
just formatting things
```suggestion
            meta = traverse_obj(
                self._download_json(
                    api_url, vod_id, 'Downloading stream url info', fatal=False, query={
                        **api_info,
                        'type': 'json',
                        'optional_id': vod_id,
                        'active_flg': 1,
                    }), ('meta', 0))

            stream_url = traverse_obj(
                meta, ('movie_url', ('mb_auto', 'auto_sp', 'auto_pc'), {url_or_none}), get_all=False)
```